CMakeLists.txt

```diff
@@ -87,6 +91,7 @@ target_compile_options(roboteam_utils PRIVATE
         )
 add_executable(utils_tests
         test/main.cpp
+        test/PubSubTest.cpp
```

Comment:
PubSub is part of a different library now. It's fine to test it here, but then you should utils_tests against networking as well.

Reply:
Will fix this when in office.